coalib/bearlib/languages/documentation/DocstyleDefinition.py

```diff
@@ -106,6 +117,20 @@ def markers(self):
         """
         return self._markers
 
+    @property
+    def metadata(self):
+        """
+        A namedtuple of certain attributes present in the documentation.
+
+        These attributes are used to define parts of the documentation.
+        """
+        escaped_metadata_dict = self._metadata._asdict()
+        unescaped_metadata_dict = {key: unescape(value)
+                                   for (key, value) in
+                                   escaped_metadata_dict.items()}
```

Comment:
Yes but for this case this is a namedtuple, so unescaping does happen within every coafile-parser, just not the stuff inside a namedtuple.